hbase-shell/src/main/ruby/shell/commands/deleteall.rb

```diff
@@ -30,12 +30,16 @@ def help
   hbase> deleteall 't1', 'r1'
   hbase> deleteall 't1', 'r1', 'c1'
   hbase> deleteall 't1', 'r1', 'c1', ts1
+  //'' means no specific column, it will deletes all cells in the row which timestamp is lower than
```

Comment:
should be `it deletes` or `will delete`, instead of  `it will deletes`.

Reply:
fix, thanks for the review and patience.  : )